cgmes/cgmes-conversion/src/test/resources/cim14/7buses.xiidm

```diff
@@ -18,7 +18,7 @@
             <iidm:load id="_FP.ANC12_EC" name="FP.ANC12" loadType="UNDEFINED" p0="240.0" q0="2.4000000953674316" bus="_FP.AND11_TN" connectableBus="_FP.AND11_TN" p="240.0" q="2.4000000953674316"/>
         </iidm:voltageLevel>
         <iidm:twoWindingsTransformer id="_FP.AND11-FTDPRA11-1_PT" name="FP.AND11-FTDPRA11-1" r="0.009999940171837807" x="0.9999989867210388" g="0.0" b="0.0" ratedU1="380.0" ratedU2="380.0" bus1="_FP.AND11_TN" connectableBus1="_FP.AND11_TN" voltageLevelId1="_FP.AND11_VL" bus2="_FTDPRA11_TN" connectableBus2="_FTDPRA11_TN" voltageLevelId2="_FTDPRA11_VL">
-            <iidm:phaseTapChanger lowTapPosition="1" tapPosition="17" regulationMode="FIXED_TAP">
+            <iidm:phaseTapChanger lowTapPosition="1" tapPosition="17" regulationMode="FIXED_TAP" regulating="false">
```

Comment:
Why do we have this redundant information regulating="false" as we already know that the regulationMode is fixed tap ?